tests/pytests/functional/formulas/test_docker.py

```diff
@@ -12,12 +12,17 @@ def _formula(saltstack_formula):
 
 @pytest.fixture(scope="module")
 def modules(loaders, _formula):
+    loaders.opts["file_roots"]["base"].append(
+        str(_formula.state_tree_path / f"{_formula.name}-{_formula.tag}")
+    )
     return loaders.modules
 
 
 @pytest.mark.skip_on_windows
 @pytest.mark.destructive_test
 def test_docker_formula(modules):
     ret = modules.state.sls("docker")
+    assert not ret.errors
+    assert not ret.failed
     for staterun in ret:
-        assert not staterun.result.failed
+        assert staterun.result
```

Comment:
```suggestion
        assert staterun.result is True
```
Anything other than a True boolean will also assert as True, but result should always be a boolean.

Reply:
But if @techhat says it's unreliable, let's not use it and really figure out why it's not reliable. It definitely should be.

Reply:
See my comment above.